src/validation/relative_lock_time.cairo

```diff
@@ -0,0 +1,111 @@
+use crate::types::transaction::Transaction;
+
+/// Validate a transaction input. If relative locktime is enabled,
+/// ensure the input's locktime is respected.
+pub fn validate_input(
```

Comment:
Move this function close to `validate_absolute_locktime`, give it a name: `validate_relative_locktime`.